nautobot/ipam/tests/test_api.py

```diff
@@ -482,6 +487,20 @@ def setUpTestData(cls):
         # needs to be enhanced to use the actual API serializers when `api=True`
         cls.validation_excluded_fields = ["status"]
 
+    def test_create_invalid_address(self):
+        """Pass various invalid inputs and confirm they are rejected cleanly."""
+        obj_perm = ObjectPermission(name="Test permission", actions=["add"])
+        obj_perm.save()
+        obj_perm.users.add(self.user)
+        obj_perm.object_types.add(ContentType.objects.get_for_model(self.model))
```

Comment:
Is there a specific reason you manually created this? Could this not be solved by `self.add_permissions("ipam.add_ipaddress")`? It appears to do exactly this:

https://github.com/nautobot/nautobot/blob/e7b5de94ff7bc1082fa03fdfddbe8457884ffdb5/nautobot/utilities/testing/views.py#L115-L124

Reply:
No reason, just ignorance on my part. :) Will change it!